### PR TITLE
kv: log leveldb and rocksdb to ceph log

### PR DESCRIPTION
**Pull request by liewegas**
This is much more convenient, and also avoids the LOG file growth/rotation
issue.`

### CODE REVIEW

File: src/kv/LevelDBStore.cc [hunk 1]
```diff
@@ -11,6 +11,35 @@ using std::string;
...
+    cct->get();
+  }
+  ~CephLevelDBLogger() {
+    cct->put();
+  }
+
+  // Write an entry to the log file with the specified format.
+  void Logv(const char* format, va_list ap) {
+    dout(1);
+    char buf[1048576];
```
Comment (yuyuyu101) [inline src/kv/LevelDBStore.cc]: Hmm, I would like to think 1M is a dangerous value to stack size? AFAR, some OS may only have several hundreds stack size.

Maybe we can declare a 1k buf, then try to iterate the whole line. If failed, turn to heap allocation? Or just truncate the log line.

Comment (liewegas) [inline src/kv/LevelDBStore.cc]: yeah, dropped it to 64k.  i did that trying to track down another bug but 
it wasn't really needed.

**Comment by tchaikov** [comment]
the build failure is not relevant and is being addressed by #6927.

**Comment by tchaikov** [comment]
tested locally,  the log printed by leveldb looks like:
```
2015-12-17 07:00:09.468848 7f7e46daf440 -1 WARNING: the following dangerous and experimental features are enabled: *
2015-12-17 07:00:10.445334 7f7e46daf440  1 leveldb: Delete type=3 #1

2015-12-17 07:00:13.181377 7feaf4366440 -1 WARNING: the following dangerous and experimental features are enabled: *
2015-12-17 07:00:13.181417 7feaf4366440 -1 WARNING: the following dangerous and experimental features are enabled: *
2015-12-17 07:00:13.181571 7feaf4366440  0 ceph version Development (no_version), process ceph-mon, pid 9822
2015-12-17 07:00:13.182461 7feaf4366440 -1 WARNING: the following dangerous and experimental features are enabled: *
2015-12-17 07:00:13.190737 7feaf4366440 10 load: jerasure load: lrc load: isa
2015-12-17 07:00:13.191237 7feaf4366440  1 leveldb: Recovering log #3
2015-12-17 07:00:13.191300 7feaf4366440  1 leveldb: Level-0 table #5: started
2015-12-17 07:00:14.330584 7feaf4366440  1 leveldb: Level-0 table #5: 841 bytes OK
2015-12-17 07:00:14.545547 7feaf4366440  1 leveldb: Delete type=3 #2

2015-12-17 07:00:14.545645 7feaf4366440  1 leveldb: Delete type=0 #3
```

lgtm

**Pull request merged**